core/controlplane/config/templates/cloud-config-controller

```diff
@@ -869,6 +869,8 @@ write_files:
                         key: etcd_endpoints
                   - name: CALICO_NETWORKING_BACKEND
                     value: "none"
+                  - name: CLUSTER_TYPE
+                    value: "kubeaws,canal"
```

Comment:
Just for my education, are there any conventions for deciding what the value should be?
For instance, [kops seems to have `kops,canal` for the value](https://github.com/kubernetes/kops/pull/3162/files), which looks consistent with this one for kube-aws.